Source/WebKit/UIProcess/API/APIFullscreenClient.h

```diff
@@ -57,6 +57,10 @@ class FullscreenClient {
 #if PLATFORM(IOS_FAMILY)
     virtual void requestPresentingViewController(CompletionHandler<void(UIViewController *, NSError *)>&&) { }
 #endif
+
+#if ENABLE(LINEAR_MEDIA_PLAYER)
+    virtual bool preventDocking(WebKit::WebPageProxy*) { return true; }
```

Comment:
Why does this return true but the default value of FullscreenClient::preventDocking is false?